omrtrace/makefile

```diff
@@ -26,8 +26,10 @@ OBJECTS := $(patsubst %.cpp,%$(OBJEXT),$(wildcard *.cpp))
 OBJECTS += $(patsubst %.c,%$(OBJEXT),$(wildcard *.c))
 
 ifeq (gcc,$(OMR_TOOLCHAIN))
-# disable builtin strncpy buffer length check
-  MODULE_CXXFLAGS+=-U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1
+# disable builtin strncpy buffer length check for optimized builds (_FORTIFY_SOURCE=1 requires -O3)
+	ifeq ($(OMR_OPTIMIZE),1)
```

Comment:
In general do our makefiles indent conditions?  If so do they use tabs or spaces?  Can you verify and fix if needed?  Thanks